src/filters/ws.rs

```diff
@@ -343,13 +343,22 @@ impl Message {
     }
 
     /// Try to get a reference to the string text, if this is a Text message.
+    #[deprecated(since = "0.3.0", note = "Use the as_text instead")]
```

Comment:
I don't think we need a deprecation note, we can just release a minor version without `to_str` to address this change :)

Reply:
Should I just remove `to_str` then?

Reply:
yeah :)